### ISSUE TITLE
Whitespace Characters

### ISSUE THREAD

**Issue by wbw20** [jonmagic/scriptular#34]
Hello

Great project.  Have you guys ever considered an option for displaying whitespace characters in the match results?  Kind of like Sublime Text does?

![screen shot 2015-01-30 at 11 47 32 am](https://cloud.githubusercontent.com/assets/833121/5982223/dc4c8dea-a875-11e4-8c27-b0f060a2d878.png)

Maybe it could be a toggle options or something?  Just a thought.  If there is interest I can work on a PR.

**Comment by jonmagic** [comment]
I'd never thought of it, seems like an interesting idea though. Can you give an example of where it would be helpful when building a regular expression?

**Comment by wbw20** [comment]
When you are matching whitespace characters it is difficult to see exactly what the capture groups have in them.  Maybe this isn't a problem for most people but it is something I was trying to do.

**Comment by jonmagic** [comment]
Makes sense to me :+1: I think this would be a good feature for the tool, I'd love to see a pull request :sparkles:

Not this week, but next week I would probably have some time to pair program too if you were interested in pairing on it at all (I usually use [Screenhero](https://screenhero.com), but it looks like they have been acquired and don't allow new signups).

**Comment by wbw20** [comment]
Hey Jon,

I had a little extra time today so I threw together a prototype.  Let me know if you like the direction it is going.  :v: 

https://github.com/wbw20/scriptular/compare/wbw20:master...show_whitespace_chars

![screen shot 2015-02-17 at 5 26 55 pm](https://cloud.githubusercontent.com/assets/833121/6240923/372703e6-b6ca-11e4-9023-3637d9d4333a.png)

**Comment by jonmagic** [comment]
@wbw20 I dig it :+1: Looks good and conveys valuable information.

**Comment by jonmagic** [comment]
Any interest in opening a PR with your changes?
